aws/cloudformation/components/database.yml.erb

```diff
@@ -2,7 +2,7 @@
     Type: AWS::RDS::DBCluster
     Properties:
       DBClusterIdentifier: !Sub "${AWS::StackName}-cluster"
-      DBClusterParameterGroupName: !Ref AuroraClusterDBParameters
+      DBClusterParameterGroupName: !Ref AuroraClusterDBParameters # Resource name generated from db_parameters.yml.erb.
```

Comment:
cloudformation _might_ remove same-line comment when it parses it. This is valid, it just might disappear when we view the template in the Cloudformation Console.